src/main/java/jenkins/plugins/office365connector/FactBuilder.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package jenkins.plugins.office365connector;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import hudson.model.User;
+import jenkins.plugins.office365connector.model.Fact;
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * Collects helper methods that create instance of {@link jenkins.plugins.office365connector.model.Fact Fact} class.
+ *
+ * @author Damian Szczepanik (damianszczepanik@github)
+ */
+public class FactBuilder {
+
+    final static String NAME_STATUS = "Status";
+    final static String NAME_REMARKS = "Remarks";
+    final static String NAME_CULPRITS = "Culprits";
+
+    private final List<Fact> facts = new ArrayList<>();
+
+    public void buildStatusStarted() {
+        facts.add(new Fact(NAME_STATUS, "Started"));
+    }
+
+    public void buildStatusRunning() {
+        facts.add(new Fact(NAME_STATUS, "Running"));
+    }
+
+    public static Fact buildStatus() {
```

Comment:
will be updated to no static later - this is just first approach to get feedback about